src/oop/oop1.py

```diff
@@ -17,3 +17,23 @@
 #     pass
 #
 # Put a comment noting which class is the base class
+class Vehicle:
```

Comment:
I know you understand this material based on what you have done this week but make sure you read all the instructions. Line 19 is asking for you to add a comment indicating which class is the base class(even if it is obvious)